src/Components/ArtistFilter/Artworks.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from "react"
+import {
+  ConnectionData,
+  createPaginationContainer,
+  graphql,
+  RelayPaginationProp,
+} from "react-relay"
+import { Subscribe } from "unstated"
+
+import { FilterState } from "./state"
+import ArtworkGrid from "../ArtworkGrid"
+
+interface Props extends RelayProps {
+  relay?: RelayPaginationProp
```

Comment:
This shouldn’t need to be optional anymore, the external props interface of the relay container won’t include/require this prop anymore with the current DefinitelyTyped typings.